### DISCUSSION TITLE
Welcome to scrolltotop Discussions!

### DISCUSSION THREAD

**Discussion by pratikabu** [pratikabu/scrolltotop#144] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scatteredpixels** [comment]
Hello — Well, it says that one should comment with an introduction of oneself, so here goes: My pseudonym is “scatteredpixels” for reasons I don't remember. I'm retired and live in the U.S.'s Pacific Northwest though I was born & raised in the Los Angeles area. I'm on the spectrum and am too wordy. I spend all my time in front of my computers and, unfortunately, don't live a balanced life.

The reason I'm here is to comment on an aspect of Scroll To Top. When a STT icon/button covers a button on the web page, one cannot activate the web page's button (please see attached). I'd like to suggest that it would be amazingly helpful if, when one holds down (e.g.) the Control key, the STT's buttons would be temporarily deactivated in order that a click would go directly to the button on the web page (the button “under” the STT icons.

Thank you!

![Cannot push trash can (delete item) button Icon](https://github.com/user-attachments/assets/34b073d1-953a-421a-9fd0-dde31077d4e8)

**Reply by pratikabu** [reply]
That's a good point, there is a provision when you use normal Scroll To Top, it has a temporary close button, but in the dual arrow mode, there isn't a way to hide it temporarily on the page.

Also, good to know about you. :)
